doc/apimanager-reference.md

```diff
@@ -790,6 +798,10 @@ For Oracle:
 | NAMESPACE | Define the namespace to be used by System's Redis Database. The empty value means not namespaced | `""` |
 | SENTINEL_HOSTS | System's Redis sentinel hosts. Used only when Redis sentinel is configured | `""` |
 | SENTINEL_ROLE | System's Redis sentinel role name. Used only when Redis sentinel is configured | `""` |
+| REDIS_CA_FILE | System's redis file that contains the Certificate Authority (CA) certificate | `""` |
+| REDIS_CLIENT_CERT | System's Redis Client certificate | `""`|
+| REDIS_PRIVATE_KEY | System's redis private key used for authentication in SSL/TLS communication | `""`|
+| REDIS_SSL | This field is "true" if any of the other fields (REDIS_CA_FILE, REDIS_CLIENT_CERT, REDIS_PRIVATE_KEY) are not emp``ty. Otherwise it is "false"| `false`|
```

Comment:
```suggestion
| REDIS_SSL | This field is "true" if any of the other fields (REDIS_CA_FILE, REDIS_CLIENT_CERT, REDIS_PRIVATE_KEY) are not empty. Otherwise it is "false"| `false`|
```
Typo

Reply:
Updated. Thank you!